app/models/thredded/messageboard.rb

```diff
@@ -50,11 +56,35 @@ class Messageboard < ActiveRecord::Base
 
     has_many :post_moderation_records, inverse_of: :messageboard, dependent: :delete_all
 
-    default_scope { where(closed: false).order(topics_count: :desc) }
-
+    default_scope { where(closed: false) }
+    # rubocop:disable Style/Lambda
     scope :top_level_messageboards, -> { where(group: nil) }
     scope :by_messageboard_group, ->(group) { where(group: group.id) }
-
+    scope :ordered, ->(order = Thredded.messageboards_order) {
+      case order
+      when :position
+        ordered_by_position
+      when :created_at_asc
+        ordered_by_created_at_asc
+      when :last_post_at_desc
+        ordered_by_last_post_at_desc
+      when :topics_count_desc
+        ordered_by_topics_count_desc
+      end.order(id: :asc)
```

Comment:
just a thought, perhaps we can always order by `position`, so here this would be

```ruby
case order
when :position
  self
when ...
  ...
end.order(position: :asc, id: :asc)
```